pkg/pfcpsim/session/far_builder.go

```diff
@@ -0,0 +1,88 @@
+package session
+
+import (
+	"github.com/wmnsk/go-pfcp/ie"
+)
+
+type farBuilder struct {
```

Comment:
using lowercase because I'd like to avoid exposing directly the builder struct

Reply:
that's fine